packages/server/src/resources/resources.mock.json

```diff
@@ -0,0 +1 @@
 № 4 ХАРКІВСЬКОЇ МІСЬКОЇ РАДИ»","address":"Харківська обл., місто Харків, Шевченківський район ВУЛИЦЯ ДВАДЦЯТЬ ТРЕТЬОГО СЕРПНЯ буд. 29"},"contactPerson":{"name":"Лев Петрович Шевчук","email":"kharkivua@gmail.com","phone":"+380631175283","website":"bodnarenko.net.ua"}},{"id":25,"name":"Витратомір кисню, 0 – 15 л/хв","quantity":5,"quantityCovered":0,"price":1384,"state":"Created","beneficiary":{"name":"КНП «Березівський районний центр первинної медико-санітарної допомоги»","address":"Одеська обл., Березівський район, місто Березівка ВУЛИЦЯ БОЛЬНИЧНА буд. 6"},"contactPerson":{"name":"Юрій Петрович Кравчук","email":"ivan.vasilcuk@gmail.com","phone":"0967806386","website":"berezivsk-odessa.org.ua"}},{"id":26,"name":"Респіратор класу захисту FFP3","quantity":13455,"quantityCovered":400,"price":134,"state":"Created","beneficiary":{"name":"Центр первинної медико-санітарної допомоги Мар'їнської районної ради","address":"Донецька обл., Мар'їнський район, місто Мар'їнка ПРОСПЕКТ ДРУЖБИ буд. 22-А"},"contactPerson":{"name":"Броваренко Ярослав Романович","email":"brovarenko1982@ukr.net","phone":"38(067)793-53-61","website":"mariinka-don.ua"}},{"id":27,"name":"Аналізатор кисню, ультразвуковий, ручний","quantity":15,"quantityCovered":1,"price":17800,"state":"Created","beneficiary":{"name":"КНП «ЦЕНТР ПЕРВИННОЇ МЕДИКО-САНІТАРНОЇ ДОПОМОГИ ЛЮБЕШІВСЬКОЇ СЕЛИЩНОЇ РАДИ»","address":"Волинська обл., Любешівський район, селище міського типу Любешів ВУЛИЦЯ НЕЗАЛЕЖНОСТІ буд. 3"},"contactPerson":{"name":"Микитюк Олександр Федорович","email":"mykitukua@gmail.com","phone":"38(067)793-53-61","website":"vubishevska-rada.ua"}},{"id":28,"name":"Концентратор кисню, 5 л/хв","quantity":17,"quantityCovered":10,"price":178901,"state":"Created","beneficiary":{"name":"КНП «Олевський Центр первинної медичної допомоги Олевської міської ради»","address":"Житомирська обл., Олевський район, місто Олевськ ВУЛИЦЯ СВЯТО-МИКОЛАЇВСЬКА буд. 46"},"contactPerson":{"name":"Григорій Сергійович Пономарчук","email":"ponomarchuuk0192@gmail.com","phone":"+380635349277","website":"zhytomyr.org.ua"}},{"id":29,"name":"Канюлі назальні, кисневі, для дорослих, одноразового використання","quantity":57,"quantityCovered":5,"price":20,"state":"Created","beneficiary":{"name":"КНП «Дрогобицька районна лікарня»","address":"Львівська обл., Дрогобицький район, село Нижні Гаї ВУЛ. ВЕРБОВА буд. 17"},"contactPerson":{"name":"Броваренко Михайло Володимирович","email":"brovarenko18972@gmail.com","phone":"+380635652370","website":"drogobytsk.ua"}},{"id":30,"name":"Ларингоскопічний набір для дорослих, дітей","quantity":700,"quantityCovered":50,"price":10780,"state":"Created","beneficiary":{"name":"Товариство з обмеженою відповідальністю \"Сімейна клініка \"МЕДЕКСПЕРТ\"","address":"Дніпропетровська обл., місто Дніпро, Центральний район ПРОСПЕКТ БОГДАНА ХМЕЛЬНИЦЬКОГО буд. 12 Г прим. 68"},"contactPerson":{"name":"Шевченко Bалерій Борисович","email":"valery.shevchenkol@gmail.com","phone":"+380634626451","website":"medexpert.ua"}}]
```

Comment:
Looks better when it's unfolded with new lines

Reply:
Got it. I'll format JSON to make it readble